### PR TITLE
Feature/upgrade-engine

### PR DESCRIPTION
**Pull request by TomKopp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,15 +50,11 @@
...
     "eslint": "8.35.0",
-    "eslint-config-react-app": "7.0.1",
     "eslint-import-resolver-typescript": "3.5.3",
-    "eslint-plugin-flowtype": "8.0.3",
     "eslint-plugin-import": "2.27.5",
-    "eslint-plugin-jsx-a11y": "6.7.1",
     "git-rev-sync": "3.0.2",
     "jest": "29.4.3",
     "nodemon": "2.0.20",
-    "npm-run-all": "4.1.5",
```
Comment (davidenke) [inline package.json]: This should not be part of this PR.

File: package.json [hunk 2]
```diff
@@ -13,6 +13,8 @@
   },
   "dependencies": {
     "@babel/core": "7.21.0",
+    "@babel/preset-env": "7.20.2",
+    "@babel/types": "7.21.2",
```
Comment (davidenke) [inline package.json]: Babel is used at runtime at some point, so we do not rely on all of it's tooling. Despite this being not relevant to this very PR, would you mind to open a follow-up PR describing the necessity of having these dependencies as well?

**Review by davidenke — CHANGES_REQUESTED** [review]

**Comment by davidenke** [comment]
Would you please pin the engine in the `.nvmrc` and `package.json` as well? Since we'll do explicit Node updates and build with tagged images, we can spec the version for the complete semver range.

File: package.json (hunk 1 above)
Comment (TomKopp) [reply package.json]: Whoops, committed that to the wrong branch

File: package.json (hunk 2 above)
Comment (TomKopp) [reply package.json]: Removed from this PR

**Comment by TomKopp** [comment]
Pined Node version in package.json and .npmrc

File: package.json (hunk 2 above)
Comment (TomKopp) [reply package.json]: See PR #12.

**Review by davidenke — APPROVED** [review]
